src/implot_visualization/run_client.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
In the `cmakeImprovements` branch, i've added a custom target, `cmake --build build --target serve` (or `make serve` in the build directory) will launch the server (and make sure everything is compiled before).

Reply:
Good idea! This will make the usage easier. Thank you!